rollup.go

```diff
@@ -16,12 +16,12 @@ import (
 
 // Renames a file with the current timestamp
 func renameFileTimestamp(cfg *Config) (string, error) {
-	t := time.Now()
+	newFileName := time.Now().UTC().Format("2006-01-02T15-04-05.00000") + ".log"
```

Comment:
Hyphens are fine. But the "T" does look a bit awkward. Its very clear enough which part is date and which part is time, so using T to designate time isn't very elegant. Since we are not following the spec to the word anyway, how about using an underscore ? It clearly denotes a different set of information and gives ample visual space between the date and time.

```
2006-01-02_15-04-05.00000
```
looks clean and simple. What do you think ?